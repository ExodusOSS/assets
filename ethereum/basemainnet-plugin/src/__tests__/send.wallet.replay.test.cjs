jest.exodus.mock.websocketReplay()

import('./send.wallet.integration.test.js')
