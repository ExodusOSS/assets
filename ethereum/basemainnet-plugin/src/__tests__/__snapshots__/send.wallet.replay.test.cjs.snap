exports[`basemainnet send transfer integration test Send all eth from first to second portfolio 1`] = `
{
  "1_amount": "0.000795780644563539 BASE",
  "2_txId": "0x77fe66df299e8a1344420702c3de1f1c09b376148af7338c96ee1bffaa6e6981",
  "3_txRawHex": "02f872822105138401312d00840d1cef0082520894f6c138c36341138ddfc314a11038da8264b7ef098702d3c21ea36a5380c001a0e5d6ef98590a7749a51fc9f5e023064edf18cff74c11b0dcca75cb80fa4f58039f2c17ec5f2f3383a523996bc93f1a3b32412db5a96aa2021902c9864342a05b",
  "4_tx": {
    "coinAmount": "-0.000795780644563539 BASE",
    "coinName": "basemainnet",
    "confirmations": 0,
    "currencies": {
      "basemainnet": {
        "BASE": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 21000,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "20000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.000004628433136375 BASE",
    "feeCoinName": "basemainnet",
    "to": "0xf6c138C36341138dDFC314a11038dA8264B7Ef09",
    "txId": "0x77fe66df299e8a1344420702c3de1f1c09b376148af7338c96ee1bffaa6e6981",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0.000795780644563539 BASE",
        "spendable": "0.000795780644563539 BASE",
        "spendableBalance": "0.000795780644563539 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.000795780644563539 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0 BASE",
        "spendable": "0 BASE",
        "spendableBalance": "0 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0.000800409077699914 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
}
`;

exports[`basemainnet send transfer integration test Send some eth from first to second portfolio 1`] = `
{
  "1_amount": "200000000000000 wei",
  "2_txId": "0x01508121ee3c17ce9244f1af3e48acf22e454e375d845194eb96ff6fedd26a06",
  "3_txRawHex": "02f872822105138401312d00840d1cef0082520894f6c138c36341138ddfc314a11038da8264b7ef0986b5e620f4800080c001a08354cdddc63f977d7b16c4edf2480c4e6b10994bf70abad55d25e10dbe6bffa8a03df4329c1131a290e3d5a1f5ece1506b95fa919a99fb5a7096e26265a9f2da13",
  "4_tx": {
    "coinAmount": "-0.0002 BASE",
    "coinName": "basemainnet",
    "confirmations": 0,
    "currencies": {
      "basemainnet": {
        "BASE": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 21000,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "20000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.000004628433136373 BASE",
    "feeCoinName": "basemainnet",
    "to": "0xf6c138C36341138dDFC314a11038dA8264B7Ef09",
    "txId": "0x01508121ee3c17ce9244f1af3e48acf22e454e375d845194eb96ff6fedd26a06",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0.000795780644563539 BASE",
        "spendable": "0.000795780644563539 BASE",
        "spendableBalance": "0.000795780644563539 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.000795780644563539 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0.000591152211427166 BASE",
        "spendable": "0.000591152211427166 BASE",
        "spendableBalance": "0.000591152211427166 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.000591152211427166 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0.000204628433136373 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
}
`;

exports[`basemainnet send transfer integration test Send some eth from first to second portfolio with provided feeAmount 1`] = `
{
  "1_amount": "200000000000000 wei",
  "2_txId": "0x01508121ee3c17ce9244f1af3e48acf22e454e375d845194eb96ff6fedd26a06",
  "3_txRawHex": "02f872822105138401312d00840d1cef0082520894f6c138c36341138ddfc314a11038da8264b7ef0986b5e620f4800080c001a08354cdddc63f977d7b16c4edf2480c4e6b10994bf70abad55d25e10dbe6bffa8a03df4329c1131a290e3d5a1f5ece1506b95fa919a99fb5a7096e26265a9f2da13",
  "4_tx": {
    "coinAmount": "-0.0002 BASE",
    "coinName": "basemainnet",
    "confirmations": 0,
    "currencies": {
      "basemainnet": {
        "BASE": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 21000,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "20000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.000001234433136375 BASE",
    "feeCoinName": "basemainnet",
    "to": "0xf6c138C36341138dDFC314a11038dA8264B7Ef09",
    "txId": "0x01508121ee3c17ce9244f1af3e48acf22e454e375d845194eb96ff6fedd26a06",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0.000795780644563539 BASE",
        "spendable": "0.000795780644563539 BASE",
        "spendableBalance": "0.000795780644563539 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.000795780644563539 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "basemainnet": {
        "balance": "0.000594546211427164 BASE",
        "spendable": "0.000594546211427164 BASE",
        "spendableBalance": "0.000594546211427164 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.000594546211427164 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0.000201234433136375 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
    "exodus_1": {
      "basemainnet": {
        "balance": "0.0002 BASE",
        "spendable": "0.0002 BASE",
        "spendableBalance": "0.0002 BASE",
        "staked": "0 BASE",
        "staking": "0 BASE",
        "total": "0.0002 BASE",
        "unconfirmedReceived": "0 BASE",
        "unconfirmedSent": "0 BASE",
        "unstaked": "0 BASE",
        "unstaking": "0 BASE",
      },
    },
  },
}
`;
